variables.tf

```diff
@@ -79,6 +79,12 @@ variable "apiary_log_prefix" {
   default     = ""
 }
 
+variable "enable_metrics" {
+  description = "Enable sending metastore metrics to CloudWatch."
```

Comment:
metastore  -> Metastore

Reply:
`Enable sending metastore metrics to CloudWatch.` --> `Enable sending Hive Metastore metrics to CloudWatch.`